site/tests/conftest.py

```diff
@@ -8,17 +8,40 @@
 """Pytest fixtures."""
 
 from collections import namedtuple
+from copy import deepcopy
 
 import pytest
-from flask_security import login_user
+from flask import g
+from flask_principal import RoleNeed
+from flask_security import login_user, logout_user
 from flask_security.utils import hash_password
+from invenio_access.models import ActionRoles
+from invenio_access.permissions import superuser_access, system_identity
+from invenio_accounts.models import Role
 from invenio_accounts.testutils import login_user_via_session
+from invenio_administration.permissions import administration_access_action
 from invenio_app.factory import create_api as _create_api
+from invenio_communities import current_communities
+from invenio_communities.communities.records.api import Community
+from invenio_pidstore.errors import PIDDoesNotExistError
+from invenio_vocabularies.proxies import current_service as vocabulary_service
+from invenio_vocabularies.records.api import Vocabulary
+
+from zenodo_rdm import config
+from zenodo_rdm.legacy.requests.record_upgrade import LegacyRecordUpgrade
 
 
 @pytest.fixture(scope="module")
 def app_config(app_config):
     """Mimic an instance's configuration."""
+    supported_configurations = ["RDM_PERMISSION_POLICY"]
+
+    for config_key in supported_configurations:
+        app_config[config_key] = getattr(config, config_key, None)
+
+    app_config["REQUESTS_REGISTERED_TYPES"] = [LegacyRecordUpgrade()]
+    app_config["REST_CSRF_ENABLED"] = False
```

Comment:
good finding !
I was having an error in my PR related to this and it was driving me a bit crazy 🤯